parser.y

```diff
@@ -237,6 +238,12 @@ rule
                         @grammar.add_parameterizing_rule(rule)
                       }
 
+  inline_declaration: "%rule" "%inline" id_colon ":" rule_rhs_list
```

Comment:
Just for clarification, this grammar rule doesn't allow parameter for inline rule. For example, "%rule %inline op(a): ..." is not allowed.
Is this intentional?
If it will be supported in following PR, it's ok for me.

Reply:
Yes. I plan to implement it in a follow-up PR. Sorry for the confusion :bow:

Reply:
Understand, no worries.